mlir/lib/Dialect/OpenACC/IR/OpenACC.cpp

```diff
@@ -1854,6 +1908,13 @@ LogicalResult acc::LoopOp::verify() {
           "reductions", false)))
     return failure();
 
+  if (getCombined().has_value() &&
+      (getCombined().value() != acc::CombinedConstructsType::ParallelLoop &&
+       getCombined().value() != acc::CombinedConstructsType::KernelsLoop &&
+       getCombined().value() != acc::CombinedConstructsType::SerialLoop)) {
```

Comment:
Is it possible that the value is not one of the enumeration?

Reply:
Not right now - but because the enum type is extensible, it made sense to be explicit about which ones are legal.

Reply:
Marking done - this was just extra defensive checking. But should be OK.